jwt/api_jwt.py

```diff
@@ -100,6 +98,18 @@ def _validate_claims(self, payload, options, audience=None, issuer=None,
                           'Please use verify_exp in options instead.',
                           DeprecationWarning)
 
+        deprecated_requires = ['require_exp', 'require_iat', 'require_nbf']
+        verify_claims = set(deprecated_requires).intersection(options)
+        require_options = options.setdefault('require', [])
+        for opt in verify_claims:
+            opt_claim = opt[8:]
+            if options[opt]:
+                require_options.append(opt_claim)
+            warnings.warn('The ' + opt + 'parameter is deprecated. '
```

Comment:
Let's do `warnings.warn('The {0} parameter is deprecated. Please add {1} to the require list in options instead.'.format(opt, opt_claim), DeprecationWarning)` instead